docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+  rest-server:
+    image: tomcat:9-jre8
+    container_name: rest-server
+    volumes:
+      - ./rest/target/rest.war:/usr/local/tomcat/webapps/rest.war
+      - logs:/usr/local/tomcat/logs
+    ports:
+      - "8081:8080"
+  rest-client:
+    image: tomcat:9-jre8
+    container_name: rest-client
+    environment:
+      - "JAVA_OPTS=-Dprofile=docker"
+    volumes:
+       - ./webapp/target/webapp.war:/usr/local/tomcat/webapps/webapp.war
+       - logs:/usr/local/tomcat/logs
+    ports:
+      - "8082:8080"
+    depends_on:
+      - "rest-server"
+
+volumes:
+  logs:
```

Comment:
would be better to create separated log directories for each application

Reply:
Ok